src/gtpu/encap.c

```diff
@@ -544,11 +564,16 @@ int check_urr(struct pdr *pdr, u64 vol, u64 vol_mbqe, bool uplink){
     u32 *triggers;
     u32 report_num = 0;
     struct urr *urr, **urrs;
-    struct user_report *report;
+    struct usage_report *report;
```

Comment:
```suggestion
    u32 *triggers = NULL;
    u32 report_num = 0;
    struct urr *urr, **urrs = NULL;
    struct usage_report *report = NULL;
```
better to set NULL
or 
```
if (report) {
        kfree(report);
 }
```
may panic